base/services/nfsd/idmap.c

```diff
@@ -369,10 +366,13 @@ static int config_init(
         goto out;
     }
 
+    GetSystemDirectoryA(config_path, ARRAYSIZE(config_path));
+    strcat(config_path, "\\drivers\\etc\\ms-nfs41-idmap.conf");
```

Comment:
Check GetSystemDirectory return and use string safe functions please

Reply:
OK.